streaming/src/main/scala/org/apache/gearpump/streaming/task/TaskActor.scala

```diff
@@ -86,36 +85,13 @@ class TaskActor(val taskContextData : TaskContextData, userConf : UserConfig, va
   def onStop() : Unit = task.onStop()
 
   def output(msg : Message) : Unit = {
-    if (null == outputTaskIds || outputTaskIds.length == 0) {
-      return
-    }
-
-    val partitions = partitioner.getPartitions(msg, taskId.index)
-
-    var start = 0
-
-    sendThroughput.mark(partitions.length)
-
-    while (start < partitions.length) {
-      val partition = partitions(start)
-
-      transport(msg, outputTaskIds(partition))
-      val ackRequest = flowControl.sendMessage(partition)
-      if (null != ackRequest) {
-        transport(ackRequest, outputTaskIds(partition))
-      }
-
-      start = start + 1
-    }
+    this.subscriptions.foreach(_._2.sendMessage(msg))
```

Comment:
It seems our model is: all subscriptions will receive the message. Inside each subscription, one task actor will receive.

Do we want to support use case like: one of all subscription receive the message. Or one task actor can generate multiple output streams?
For example, for IoT data flow in, if the data exceeds some threshold, generate  alert message to one stream (subscription), otherwise generate data to a normal stream (subscription).